inst/report/scripts/viz_utils.R

```diff
@@ -0,0 +1,922 @@
+# Author(s): Awa Synthia
+# Last modified: 2024
+
+# Load necessary packages
+library(dplyr)
+library(stringr)
+library(visNetwork)
+library(DT)
+library(plotly)
+
+# Function to generate the InterProScan Visualization
+generate_ipr_genes_visualization <- function(data, app_data,
```

Comment:
Rename function to reflect how this is different from `plotIPR2Viz`?
https://jravilab.github.io/MolEvolvR/reference/index.html

Reply:
renamed all functions in the viz utils.